core/unit_test/incremental/Test06_ParallelFor_MDRangePolicy.hpp

```diff
@@ -195,7 +195,7 @@ struct TestMDRangePolicy {
     Kokkos::parallel_for("MDRange3D", mdPolicy_3D, Functor_3D);
 
     // Copy the data back to Host memory space
-    Kokkos::Impl::DeepCopy<d_memspace_type, h_memspace_type>(
+    Kokkos::Impl::DeepCopy<h_memspace_type, d_memspace_type>(
```

Comment:
caught by OpenMPTarget testing, since it is the only one where the memcpy operation doesn't deduce the direction from the pointers, and thus you have to provide the correct memory spaces here ...